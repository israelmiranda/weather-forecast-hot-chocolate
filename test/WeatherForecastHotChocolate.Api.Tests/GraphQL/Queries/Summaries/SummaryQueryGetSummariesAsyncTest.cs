using Xunit;

namespace WeatherForecastHotChocolate.Api.Tests.GraphQL.Queries.Summaries
{
    public class SummaryQueryGetSummariesAsyncTest : IClassFixture<TestFixture>
    {
        [Fact]
        public void Get_Summaries_Returns_Async()
        {
            // arrange

            // act

            // assert
        }
    }
}
