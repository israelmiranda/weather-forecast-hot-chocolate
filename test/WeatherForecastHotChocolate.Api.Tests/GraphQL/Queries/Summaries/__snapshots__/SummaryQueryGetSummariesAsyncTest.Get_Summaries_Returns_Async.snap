{
  "data": {
    "summaries": [
      {
        "id": "71a631ca78fd48569740fe69a78ce456",
        "description": "SummaryTest"
      }
    ]
  }
}
